test/conftest.py

```diff
@@ -35,6 +36,15 @@ def client():
     return TestClient(app)
 
 
+@pytest.fixture
+def event_loop():
+    """Create an instance of the default event loop for each test case"""
```

Comment:
Maybe we can add a description why we need to insert an event loop to each test case and paste the bug URL.

Reply:
Updated the docstring.